vms/platformvm/network/network_test.go

```diff
@@ -329,44 +302,11 @@ func TestNetworkIssueTx(t *testing.T) {
 			)
 			require.NoError(err)
 
-			err = n.IssueTx(context.Background(), tx)
+			err = nIntf.IssueTxFromRPC(tx)
 			require.ErrorIs(err, tt.expectedErr)
+
+			n := nIntf.(*network)
```

Comment:
Might make more sense to just remove the interface

Reply:
Done